packages/rpc-core/src/rpc-methods/getInflationGovernor.ts

```diff
@@ -0,0 +1,30 @@
+import { Commitment, U64UnsafeBeyond2Pow53Minus1 } from './common';
+
+type GetInflationGovernorApiResponse = Readonly<{
+    /** The initial inflation percentage from time 0 */
+    initial: number; // Until we land on best type for `f64`
+    /** Percentage of total inflation allocated to the foundation */
+    foundation: number; // Until we land on best type for `f64`
+    /** Duration of foundation pool inflation in years */
+    foundationTerm: U64UnsafeBeyond2Pow53Minus1; // Until we land on best type for `f64`
```

Comment:
I'd just use `number` here too TBH. If it's f64 on the backend then we should support an f64 (as best as we can) even if the public networks always use an int.

Reply:
I forgot to remove that comment but that one is `u64`!

Reply:
nvm I must have misread it, it's `f64`. Thought it was `u64`!